prospero-common/src/main/java/org/wildfly/prospero/test/MetadataTestUtils.java

```diff
@@ -0,0 +1,44 @@
+package org.wildfly.prospero.test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.eclipse.aether.repository.RemoteRepository;
+import org.wildfly.channel.Channel;
+import org.wildfly.channel.Stream;
+import org.wildfly.prospero.api.InstallationMetadata;
+import org.wildfly.prospero.api.exceptions.MetadataException;
+import org.wildfly.prospero.model.ChannelRef;
+
+public final class MetadataTestUtils {
+
+    private MetadataTestUtils() {
+    }
+
+    public static Channel createManifest(Collection<Stream> streams) {
+        return new Channel("manifest", null, null, null, streams);
+    }
+
+    public static InstallationMetadata createInstallationMetadata() throws IOException, MetadataException {
+        return createInstallationMetadata(createManifest(null), Collections.emptyList(), Collections.emptyList());
+    }
+
+    public static InstallationMetadata createInstallationMetadata(List<ChannelRef> channels,
+            List<RemoteRepository> repositories) throws IOException, MetadataException {
+        return createInstallationMetadata(createManifest(null), channels, repositories);
+    }
+
+    public static InstallationMetadata createInstallationMetadata(Channel manifest, List<ChannelRef> channels,
+            List<RemoteRepository> repositories) throws MetadataException, IOException {
+        Path installation = Files.createTempDirectory("installation");
```

Comment:
should that be `.installation` -  `InstallationMetadata.METADATA_DIR`

Reply:
Right, I copy pasted that bit and didn't notice.

Reply:
Allthough, I'm not sure. The parameter is a prefix string for a temp directory. I think it shouldn't start with ".", that would sort of hide it from the developer, in case the directory was not deleted for any reason (like tests terminated with a signal).